service/start.go

```diff
@@ -91,6 +91,7 @@ func (svc *service) startNostr(ctx context.Context) error {
 			waitToReconnectSeconds = 0
 
 			svc.nip47Service.StartNotifier(relay.Context(), relay)
+			svc.nip47Service.StartNip47InfoPublisher(relay.Context(), relay, svc.lnClient)
```

Comment:
We can add a context.WithCancel and call it where we set `publishInfoEvents = false` but I think it is fine

Reply:
I think we can rewrite this code now to remove the `publishInfoEvents` boolean and only once on startup add the entries to the queue. Does that make sense?

Reply:
> We can add a context.WithCancel and call it where we set publishInfoEvents = false but I think it is fine

Sorry, I don't understand what you mean here. Why do we need it? the publisher already listens to `ctx.Done` which will be consumed when the relay context is cancelled

Reply:
Yes, but once the publish is done, we can cancel the context to stop the publisher right? Since we pass the relay context to the publisher and it stays waiting for a event to publish without cancelling. (Unlike Notifier which needs to keep listening)

Reply:
Ah ok, thanks